src/openllm/local.py

```diff
@@ -16,6 +28,7 @@ def _get_serve_cmd(bento: BentoInfo, port: int = 3000):
 
 
 def serve(bento: BentoInfo, port: int = 3000):
+    prep_env_vars(bento)
     venv = ensure_venv(bento)
     cmd, env, cwd = _get_serve_cmd(bento, port=port)
```

Comment:
It seems the prepared env isn't applied to `bentoml serve`? @rickzx

Reply:
It is applied to `bentoml serve`. The `run_command` function explicitly copies the env vars from the current process to the new process (even if it doesn't, the default subprocess will also inherit env from the current process)